examples/spec/cassandra.ini

```diff
@@ -0,0 +1,1978 @@
+[]
+mountpoint = cassandra.ini
+infos/plugins = dump
```

Comment:
dump format for an ini file?

Reply:
I think I copied this from somewhere here, it has actually no effect I think

Reply:
Actually this works better than the ini plugin itself. There is currently a crucial problem for the ini plugin #2467

Reply:
I think the problem is more that the file extension `.ini` suggest an actual INI file.

Reply:
It actually takes the storage format from infos/plugins and not from the extension

Reply:
Yes,that is exactly the point you are creating a `dump` file with `.ini` extension. That will confuse users when they try to look at the file in an external editor.

Change the plugin to one of the ini plugins (`ni` works quite well) or change the extension to `.ecf` or something else that doesn't suggest a common file format.

Reply:
For the LCDproc files we should use "ini" so that old config files will still work (or switch to YAML if @sanssecours is ready, but then we need some migration path).

Reply:
If we use the ini format then most things will not work because of multiline descriptions.

Reply:
Yes, it is a bug to be fixed. When using dump, people will not be able to edit files by hand, which is a much bigger usability problem. (It breaks their current workflow)